pillar.example

```diff
@@ -87,3 +87,12 @@ proftpd:
     ClamAV: '"on"'
     # Specify the UNIX Local Socket
     ClamLocalSocket: /var/run/clamav/clamd.sock
+    #Use it to manage modules by salt
+    override_default_modules: False
```

Comment:
One idea: not use any parameter and just verify with `salt['pillar.get']` or `salt['config.get']` presence of Modules and activate the state or not depending of this presence

Reply:
Ok I delete this parameter and i test Modules pillar in the init.sls file